packages/cli/src/cli/cogify/action.cog.ts

```diff
@@ -147,6 +165,18 @@ export class ActionCogCreate extends CommandLineAction {
     }
 
     if (expectedTiffs.size === 0) {
+      if (job.processingId != null) {
+        // Update job status if this is the processing job.
+        const jobConfig = await Config.ProcessingJob.get(job.processingId);
+        if (jobConfig != null) {
+          const jobComplete = jobConfig as ProcessingJobComplete;
+          jobComplete.status = 'complete';
+          jobComplete.projection = job.tileMatrix.projection.code;
```

Comment:
this should be set well before the end?

Also we should use `tileMatrix` over `projection`